### ISSUE TITLE
Refactor private headers install location

### ISSUE THREAD

**Issue by fpesari** [sadko4u/lsp-dsp-lib#13]
Hello,

right now `make install` installs three directories (and their contents and parent directories) to the user's computer:

```
/usr/include/lsp-plug.in/dsp
/usr/include/private/dsp
/usr/include/private/utest
```

This is not very convenient, because `/usr/include/private` does not exist on most systems and since it has a generic name, it certainly can't be owned by a lsp-dsp-lib package.

If private headers need to be installed to the user's system, I think they should all go in the LSP main directory, like this:

```
/usr/include/lsp-plug.in/dsp
/usr/include/lsp-plug.in/private/dsp
/usr/include/lsp-plug.in/private/utest
```

This is also better for packagers, as a package owning a generic path like `/usr/include/private` might be undesirable.

**Comment by sadko4u** [comment]
Private headers should not be installed at all, that's a bug.

**Comment by sadko4u** [comment]
No, I see that makefile properly handles all headers and installs only files from `include/lsp-plug.in` subdirectory.
It seems that your scripts don't handle the rule that files only from `include/lsp-plugi.in` should be installed.

**Comment by sadko4u** [comment]
Wrong. Can confirm that `private` path is already installed. Hmm.

**Comment by sadko4u** [comment]
Pushed changes to **github-issue-13** branch: d5ea347b319fd3fee7dcdff19abb18a1a6db550a
Please verify.

**Comment by fpesari** [comment]
Yes, no private headers are included now.

**Comment by sadko4u** [comment]
Merged changes into **devel** branch: 2289e11250bd4b76b46b3e7f0f4588d165a0b900
Removed **github-issue-13** branch

**Comment by sadko4u** [comment]
Released 0.5.8 version: 54d11845f04d9c5c958d0ac1d749fdc798c45f7c
